content/en/blog/_posts/2018-10-08-support-for-azure-vmss.md

```diff
@@ -65,7 +65,7 @@ In addition to the above interfaces, the storage services from the cloud provide
 
 ## Azure cloud provider implementation and VMSS
 
-In the Azure cloud provider, for every type of cluster we implement, there is a VMType option which we specify. In case of VMSS, the VM type is “vmss”.  The provisioning software (acs-engine, in future AKS etc.) would setup these values in /etc/kubernetes/azure.json file. Based on this type, various implementations would get instantiated [[3]](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/azure/azure_vmss.go)
+In the Azure cloud provider, for every type of cluster we implement, there is a VMType option which we specify. In case of VMSS, the VM type is “vmss”.  The provisioning software (acs-engine, in future AKS etc.) would setup these values in /etc/kubernetes/azure.json file. Based on this type, various implementations would get instantiated [[3]](https://github.com/kubernetes/kubernetes/blob/feature-serverside-apply/pkg/cloudprovider/providers/azure/azure_vmss.go)
```

Comment:
I would avoid linking to a feature branch here. When it's eventually removed the link will break again. Additionally, the cloud-provider code base has progressively been moved out of tree (azure has been relocated here: https://github.com/kubernetes-sigs/cloud-provider-azure)

In this instance, I would link to either the last commit that has `azure_vmss.go` or link to the root of the cloud-provider-azure directory.

Reply:
Well done. Thanks for your advice.